ttnn/cpp/ttnn/operations/data_movement/fill_pad/device/fill_pad_device_operation_types.hpp

```diff
@@ -0,0 +1,24 @@
+// SPDX-FileCopyrightText: © 2025 Tenstorrent AI ULC
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include "ttnn/tensor/tensor.hpp"
+
+namespace ttnn::operations::data_movement::fill_pad {
+
+struct operation_attributes_t {
+    float fill_value;
```

Comment:
Please add value-initialization to struct fields with primitive types.
```suggestion
    float fill_value{};
```